src/components/MainContent.js

```diff
@@ -1,13 +1,68 @@
-import { Heading } from "@chakra-ui/layout";
+import { Heading, HStack, VStack, Text, Box } from '@chakra-ui/layout';
+import { Image } from '@chakra-ui/image';
+import SideBar from './SideBar';
+import { useAuthContext } from './AuthContextProvider';
+import womanImage from '../assets/woman-image.jpg';
+import { Link } from 'react-router-dom';
 
-const MainContent = ({setActiveOption}) => {
+console.log(womanImage);
+const MainContent = () => {
+  const { isLoggedIn } = useAuthContext();
   return (
-    {
-        !isLoggedIn ?
-        <Heading>User is logged in</Heading> 
-        :
-        <Heading>Choose item in menu</Heading> 
-    }
+    <HStack>
+      {isLoggedIn ? (
```

Comment:
We shouldn't need this check here, it should just redirect to /login if the user is not logged in